charts/argus/templates/_helpers.tpl

```diff
@@ -142,6 +142,13 @@ Argus proxy details or not, for this we're using Lookup function in helm.
 {{- define "lm-credentials-and-proxy-details" -}}
 {{- $secretObj := (lookup "v1" "Secret" .Release.Namespace .Values.global.userDefinedSecret) | default dict }}
 {{- $secretData := (get $secretObj "data") | default dict }}
+{{- if and .Values.global.userDefinedSecret (not $secretData.accessID) }}
+{{- required "A valid accessID key/value is required in the provided secret" $secretData.accessID }}
```

Comment:
In the messages can we avoid mentioning key/value? A message simpler like A valid accessId is required ... Would be better.

Reply:
noted

Reply:
@rohitphatak Moved this redundant validation to lmutil chart. Kindly review #267